src/components/WrappedActionMenu/RewardsTab/TimeRemainingTooltip/TimeRemainingTooltip.tsx

```diff
@@ -60,7 +61,7 @@ export const CountDown: React.FC<Omit<ITimeRemainingTooltipInterface, 'icon' | '
       <Typography className={classes.title}>
         {displayTimeRemaining()}
       </Typography>
-      {hint}
+      <p style={{ margin: 0, color: colors.navy.lightGrey }}>{hint}</p>
```

Comment:
to z tym kolorem robisz chyba wszędzie tam, gdzie używamy tooltipów, może warto to przenieść do MobileTooltip, style klasy tooltip?